src/Elastic.Markdown/Links/CrossLinks/CrossLinkResolver.cs

```diff
@@ -51,155 +50,172 @@ public static bool TryResolve(
 	)
 	{
 		resolvedUri = null;
-		var lookup = fetchedCrossLinks.LinkReferences;
-		if (crossLinkUri.Scheme != "asciidocalypse" && lookup.TryGetValue(crossLinkUri.Scheme, out var linkReference))
-			return TryFullyValidate(errorEmitter, uriResolver, fetchedCrossLinks, linkReference, crossLinkUri, out resolvedUri);
 
-		// TODO this is temporary while we wait for all links.json to be published
-		// Here we just silently rewrite the cross_link to the url
-
-		var declaredRepositories = fetchedCrossLinks.DeclaredRepositories;
-		if (!declaredRepositories.Contains(crossLinkUri.Scheme))
+		if (crossLinkUri.Scheme.Equals("asciidocalypse") || !fetchedCrossLinks.LinkReferences.TryGetValue(crossLinkUri.Scheme, out var sourceLinkReference))
```

Comment:
I think its time to remove this leniency. All repositories are now publishing links.json files for a long while.